wgsl/index.bs

```diff
@@ -5537,6 +5673,8 @@ When an identifier is used as a [=syntax/callable=] item, it is one of:
   <dfn for=syntax>singular_expression</dfn> :
 
     | [=syntax/primary_expression=] [=syntax/postfix_expression=] ?
+
+    | [=syntax/const_literal=]
```

Comment:
This moved from `primary_expression` to `singular_expression` because otherwise it would be possible to parse the `const_literal` from `primary_expression` and a `period` from `postfix_expression`. `singular_expression` is the only caller of `primary_expression`.